app/src/main/java/com/locoquest/app/MainActivity.kt

```diff
@@ -181,7 +183,7 @@ class MainActivity : AppCompatActivity(), OnMapReadyCallback {
                 }
                 .addOnFailureListener(this) { e ->
                     // No Google Accounts found. Just continue presenting the signed-out UI.
-                    Log.d(TAG, e.localizedMessage)
+                    e.localizedMessage?.let { it1 -> Log.d(TAG, it1) }
```

Comment:
Used let to allow this variable to be called safely, as it could be null-able
https://kotlinlang.org/docs/null-safety.html#safe-calls

Reply:
Is e nullable here?

If so, I'd expect an error/warning in the original "e.localizedMessage"

worst case, if it is, that statement will just return null.  With the suggested alternative, there will be no message at all.  So... they feel about equivalent.